etl/steps/data/garden/papers/2023-01-04/farmer_lafond_2016.meta.yml

```diff
@@ -0,0 +1,420 @@
+dataset:
+  channel: garden
```

Comment:
Out of curiosity does this `channel` is used just to identify the YAML file or is there other use of it in the ETL?

Reply:
Hmm, I also wonder why it's there. We've started adding `channel` to metadata **automatically** when you call `dataset.save()`. AFAIK there's no need to set it manually.

Reply:
I suppose this is something that `etl-metadata-export` did. I will remove it from the yaml file, given that it's unnecessary.